recording/lib/storage/file.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Membrane.RTC.Engine.Endpoint.Recording.Storage.File do
+  @moduledoc """
+  `Membrane.RTC.Engine.Endpoint.Recording.Storage` implementation that saves the stream to files locally.
+  """
+  @behaviour Membrane.RTC.Engine.Endpoint.Recording.Storage
+
+  @impl true
+  def get_sink(opts) do
```

Comment:
Here we have `opts` but in `save_object` we have `output_dir` and `filename`. I would choose one option